common/spec/dependabot/dependabot_error_spec.rb

```diff
@@ -0,0 +1,217 @@
+# typed: false
+# frozen_string_literal: true
+
+require "spec_helper"
+require "dependabot/errors"
+
+RSpec.describe Dependabot::DependabotError do
+  describe Dependabot::DependabotError do
```

Comment:
Since this is testing the top-level `Dependabot::DependabotError` which contains the other namespaces, I think we should drop the nesting for these tests, having two describes with the same class is a little uncommon, this should instead just follow the namespace hierarchy 1:1.

Reply:
resolved this issue